### ISSUE TITLE
OneStatementPerLine: false match with try-with-resources

### ISSUE THREAD

**Issue by philipa** [checkstyle/checkstyle#2211]
According to discussion on #64, the following should by allowed by OneStatementPerLine:

``` java
        try (
            OutputStream fos = new FileOutputStream("foo");
            OutputStream fos2 = new FileOutputStream("bah");) {
        }
```

It isn't.
```
$ java -jar checkstyle-6.11-SNAPSHOT-all.jar -c config.xml 3.java
Starting audit...
/var/tmp/3.java:9:60: error: Only one statement per line allowed.
Audit done.
Checkstyle ends with 1 errors.

$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
    "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
    "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
  <module name="TreeWalker">
    <module name="OneStatementPerLine"/>
  </module>
</module>


 $ cat 3.java 
package test;

public class Foo {

    void bah() { 

        try (
            OutputStream fos = new FileOutputStream("foo");
            OutputStream fos2 = new FileOutputStream("bah");) {
        }


     }
}
````

**Comment by romani** [comment]
fix is merged.
